lib/puppetserver/ca/action/list.rb

```diff
@@ -96,8 +96,12 @@ def output_certs(certs)
           end
 
           certs.each do |cert|
+            # In newer versions of the CA api we return subjcet_alt_names
+            # in addition to dns_alt_names, this field includes DNS alt
+            # names but also IP alt names.
+            alt_names = cert["subject_alt_names"] || cert["dns_alt_names"]
             @logger.inform "    #{cert["name"]}".ljust(padded + 6) + " (SHA256) " + " #{cert["fingerprints"]["SHA256"]}" +
-                               (cert["dns_alt_names"].empty? ? "" : "\talt names: #{cert["dns_alt_names"]}")
+                               (alt_names.empty? ? "" : "\talt names: #{alt_names}")
```

Comment:
@Magisus 	I updated this wrt my prior comment. Let me know what you think. Otherwise I'm +1

Reply:
Good, this seems more robust.